tests/e2e/nvidia-container-toolkit_test.go

```diff
@@ -136,53 +117,52 @@ var _ = Describe("docker", Ordered, ContinueOnFailure, func() {
 	// The following should all produce the same result.
 	When("Running the cuda-deviceQuery sample", Ordered, func() {
 		BeforeAll(func(ctx context.Context) {
-			_, _, err := r.Run("docker pull nvcr.io/nvidia/k8s/cuda-sample:devicequery-cuda12.5.0")
+			_, _, err := runner.Run("docker pull nvcr.io/nvidia/k8s/cuda-sample:devicequery-cuda12.5.0")
```

Comment:
Since this string is referenced in multiple places, can we move it to a constant string var?

```
nvcr.io/nvidia/k8s/cuda-sample:devicequery-cuda12.5.0
```

Reply:
Done